app/serializers/v1/task_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module V1
+  class TaskSerializer < ActiveModel::Serializer
+
+    attributes :name, :description, :project_id
```

Comment:
It would probably be good to include the task `state` as well as that may be important to a user of the API.